[2025-08-19T13:13:02.599+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_data.generate_fraud_data manual__2025-08-19T13:11:53.103496+00:00 [queued]>
[2025-08-19T13:13:02.622+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_data.generate_fraud_data manual__2025-08-19T13:11:53.103496+00:00 [queued]>
[2025-08-19T13:13:02.624+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-19T13:13:02.659+0000] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): generate_fraud_data> on 2025-08-19 13:11:53.103496+00:00
[2025-08-19T13:13:02.671+0000] {standard_task_runner.py:60} INFO - Started process 3469 to run task
[2025-08-19T13:13:02.678+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'generate_data', 'generate_fraud_data', 'manual__2025-08-19T13:11:53.103496+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmpp5o6wxox']
[2025-08-19T13:13:02.682+0000] {standard_task_runner.py:88} INFO - Job 49: Subtask generate_fraud_data
[2025-08-19T13:13:02.844+0000] {task_command.py:423} INFO - Running <TaskInstance: generate_data.generate_fraud_data manual__2025-08-19T13:11:53.103496+00:00 [running]> on host 1205caf1711b
[2025-08-19T13:13:03.037+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='generate_data' AIRFLOW_CTX_TASK_ID='generate_fraud_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-19T13:11:53.103496+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-19T13:11:53.103496+00:00'
[2025-08-19T13:13:04.373+0000] {logging_mixin.py:188} INFO - Labeled transaction data inserted successfully.
[2025-08-19T13:13:04.407+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-08-19T13:13:04.929+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=generate_data, task_id=generate_fraud_data, execution_date=20250819T131153, start_date=20250819T131302, end_date=20250819T131304
[2025-08-19T13:13:04.985+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-19T13:13:05.045+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
